test/navigation/TrusteeOrganisationNavigatorSpec.scala

```diff
@@ -18,85 +18,201 @@ package navigation
-      val answers = emptyUserAnswers
-        .set(AddressYesNoPage(index), false).success.value
+      "Do you know address page -> YES -> Is address in UK page" in {
+        val answers = baseAnswers
+          .set(AddressYesNoPage(index), true).success.value
 
-      navigator.nextPage(AddressYesNoPage(index), fakeDraftId, answers)
-        .mustBe(CheckDetailsController.onPageLoad(index, draftId))
-    }
+        navigator.nextPage(AddressYesNoPage(index), fakeDraftId, answers)
+          .mustBe(AddressUkYesNoController.onPageLoad(index, fakeDraftId))
+      }
 
-    "Is address in UK page -> YES -> UK address page" in {
-      val answers = emptyUserAnswers
-        .set(AddressUkYesNoPage(index), true).success.value
+      "Do you know address page -> NO -> Check your answers page" in {
+        val answers = baseAnswers
+          .set(AddressYesNoPage(index), false).success.value
 
-      navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
-        .mustBe(UkAddressController.onPageLoad(index, fakeDraftId))
-    }
+        navigator.nextPage(AddressYesNoPage(index), fakeDraftId, answers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
 
-    "Is address in UK page -> NO -> International address page" in {
-      val answers = emptyUserAnswers
-        .set(AddressUkYesNoPage(index), false).success.value
+      "Is address in UK page -> YES -> UK address page" in {
+        val answers = baseAnswers
+          .set(AddressUkYesNoPage(index), true).success.value
 
-      navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
-        .mustBe(InternationalAddressController.onPageLoad(index, fakeDraftId))
-    }
+        navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
+          .mustBe(UkAddressController.onPageLoad(index, fakeDraftId))
+      }
+
+      "Is address in UK page -> NO -> International address page" in {
+        val answers = baseAnswers
+          .set(AddressUkYesNoPage(index), false).success.value
+
+        navigator.nextPage(AddressUkYesNoPage(index), fakeDraftId, answers)
+          .mustBe(InternationalAddressController.onPageLoad(index, fakeDraftId))
+      }
+
+      "UK address page -> Check your answers page" in {
+        navigator.nextPage(UkAddressPage(index), fakeDraftId, baseAnswers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
+
+      "International address page -> Check your answers page" in {
+        navigator.nextPage(InternationalAddressPage(index), fakeDraftId, baseAnswers)
+          .mustBe(CheckDetailsController.onPageLoad(index, draftId))
+      }
 
-    "UK address page -> Check your answers page" in {
-      navigator.nextPage(UkAddressPage(index), fakeDraftId, emptyUserAnswers)
-        .mustBe(CheckDetailsController.onPageLoad(index, draftId))
     }
+    "A 5mld Trust" must {
+
+      val baseAnswers = emptyUserAnswers.copy(is5mldEnabled = true)
+
```

Comment:
Missing unit tests for 5mld nav:
Name page -> UTR yes/no page
UTR yes/no page -> Yes -> UTR page
UTR yes/no page -> No -> Country of residency yes/no page